setup.py

```diff
@@ -41,6 +41,26 @@
     # style check
     'flake8>=3.7.7,<4',
     'isort>=4.3.4,<5',
+    'dlint>=0.11.0,<0.12',  # code security addon for flake8
+    'flake8-debugger>=4.0.0,<4.1',
+    'flake8-mock>=0.3,<0.4',
+    'flake8-mutable>=1.2.0,<1.3',
+    'flake8-absolute-import>=1.0,<2',
+    'flake8-multiline-containers>=0.0.18,<0.1',
+    'flake8-print>=4.0.0,<4.1',
+    'flake8-quotes>=3.3.0,<4',
+    'flake8-fixme>=1.1.1,<1.2',
+    'flake8-expression-complexity>=0.0.9,<0.1',
+    'flake8-eradicate>=1.1.0,<1.2',
+    'flake8-builtins>=1.5.3,<1.6',
+    'flake8-variables-names>=0.0.4,<0.1',
+    'pandas-vet>=0.2.2,<0.3',
+    'flake8-comprehensions>=3.6.1,<3.7',
+    'dlint>=0.11.0,<0.12',
+    'flake8-docstrings>=1.5.0,<2',
+    'pydocstyle>=6.1.1,<6.2',
```

Comment:
Since `flake8-docstrings` is the plugin is implemented using `pydocstyle`, which ends up bringing the same functionality to our libraries, we decided to drop `pydocstyle` on the following PR:  https://github.com/sdv-dev/DeepEcho/pull/48 

We can remove `pydocstyle` from here and just call `flake8` without ignoring `D` (see coments below).